Source/SwiftLintFramework/Models/RuleList+Documentation.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  RuleList+Documentation.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 8/24/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+
+extension RuleList {
+    public func generateDocumentation() -> String {
```

Comment:
I'd prefer to keep it outside `SwiftLintFramework`, but I think it's a reasonable compromise.